include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -19,6 +19,10 @@
 #include <iterator>
 #include <type_traits>
 
+#if _ONEDPL_CPP20_SPAN_PRESENT
+#include <span>
```

Comment:
I don't see `std::span` usage in this file.

Reply:
This was fixed in a consequent PR.